packages/core/src/http-client/http-agent.ts

```diff
@@ -29,10 +29,12 @@ export function getAgentConfig(
   const agentType = destination.proxyConfiguration
     ? AgentType.PROXY
     : AgentType.DEFAULT;
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  const certificateOptions = getCertificateOption(destination);
   if (agentType === AgentType.PROXY) {
-    return createProxyAgent(destination);
+    return createProxyAgent(destination,certificateOptions);
   }
-  return createDefaultAgent(destination);
+  return createDefaultAgent(destination,certificateOptions);
```

Comment:
@marikaner , I found missing spaces;) I guess the lint config should be aligned. Have we reached some decision?

Reply:
I'll create a ticket to solve this ;)

Reply:
I can not find the ticket? Is it in our backlog or in this repo?

Reply:
https://github.wdf.sap.corp/MA/scp-rocket/issues/1768